spring-cloud-config-sample/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Why is this needed?

Reply:
Otherwise Testcontainers is logging in DEBUG mode, which leads to a lot of debug logs from docker